scripts/appstream.prov

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# Transform appdata/metainfo xml file into RPM appstream(filename) provides
+#
+# Author: Michael Schroeder <mls@suse.de>
+# Based on other provides scripts from RPM
+
+OLD_IFS="$IFS"
+while read instfile ; do
+	case "$instfile" in
+	*.appdata.xml)
+		echo "appstream()"
+		echo "appstream(${instfile##*/appdata/})"
+		;;
+	*.metainfo.xml)
+		echo "appstream()"
+		echo "appstream(${instfile##*/metainfo/})"
```

Comment:
`*.metainfo.xml` is being used for everything now. You can see this in new KDE AppStream files, and GNOME ones as well. They aren't called `*.appdata.xml` anymore.